### PR TITLE
Usage Docs

### PR DESCRIPTION
**Pull request by khorne3**
The beginnings of a usage guide for FlareBot. Please let me know if:

- This shouldn't be on the hacktoberfest branch
- Any additions/corrections you'd like

## Added
/docs

## Changed
README.md

## Deprecated
N/A

## Removed
N/A

## Fixed
N/A

## Security
N/A

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Flarebot
+
+FlareBot is a Discord bot that plays music and handles basic server admin tasks. Here's how to get up and running.
+
+## Getting Started
+
+We're assuming that you have a Discord server already; if not, spin one up before proceeding.
+
+Once your server is ready, navigate go the [FlareBot homepage](https://flarebot.stream/). Click **Invite**.
```
Comment (binaryoverload) [inline docs/usage.md]: Change `navigate go` to `go to`

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+![](media/added.png)
+
+If you're interested in the music features of FlareBot, you'll need to have at least one Voice Channel. If not, create one by clicking the **plus icon** next to **VOICE CHANNELS** in the left-hand navigation pane.
+
+Set the name and channel type, and click **Create Channel** to proceed.
+
+![](media/create-voice-channel.png)
+
+At this point, you can begin using FlareBot. You can see a list of commands available to you by running **_help**. You can also see the list on the [FlareBot site](https://flarebot.stream/). There are three categories of commands: **General**, **Moderation**, and **Music**.
```
Comment (binaryoverload) [inline docs/usage.md]: Point the link to the [commands page.](https://flarebot.stream/#commands)

File: docs/usage.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+![](media/help.png)
+
+As an example, let's create a playlist that includes Of Monsters and Men's *Wolves Without Teeth* and Dolly Parton's **Jolene**. We can do so with the following two commands:
```
Comment (binaryoverload) [inline docs/usage.md]: Either have both song titles in bold or italic, not a mix of both

**Comment by khorne3** [comment]
@binaryoverload Fixed. Thank you!

File: docs/usage.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Flarebot
```
Comment (WalshyDev) [inline docs/usage.md]: Capitalise the B on bot
FlareBot

Comment (binaryoverload) [reply docs/usage.md]: Now who is picky :eyes:

Comment (WalshyDev) [reply docs/usage.md]: 👀

**Pull request merged**